Classes/BITCrashCXXExceptionHandler.mm

```diff
@@ -58,7 +58,7 @@
 
 @implementation BITCrashUncaughtCXXExceptionHandlerManager
 
-extern "C" void LIBCXXABI_NORETURN __cxa_throw(void *exception_object, std::type_info *tinfo, void (*dest)(void *))
```

Comment:
Right, I only built with Xcode 9 and it indeed fails with Xcode 8.3. I updated my PR and use `__attribute__((noreturn))` now this makes it compile with Xcode 8.3 and Xcode 9.